src/component/Legend.tsx

```diff
@@ -50,7 +50,7 @@ export type Props<TValue, TID> = DefaultProps<TValue, TID> & {
     right?: number;
   };
   payloadUniqBy?: UniqueOption<TValue, TID>;
-  onBBoxUpdate?: (box: ClientRect | DOMRect | null) => void;
+  onBBoxUpdate?: (box: DOMRect | null) => void;
```

Comment:
FYI: `ClientRect` seems to be a pre-standard version of the canonical `DOMRect`:

https://bugs.chromium.org/p/chromium/issues/detail?id=719246